modules/upgrade/pages/server-y-yast.adoc

```diff
@@ -36,11 +36,18 @@ For more information, see xref:upgrade:server-z.adoc[].
 // I've removed the graphical option, because we tell people to use a text console. LKB 2019-08-21
 . Select the appropriate target, and follow the prompts to complete the migration.
 . Reboot the server.
-. When rebooted the spacewalk services are not running until you have migrated the PostgreSQL database to version{nbsp}12.
-. Log in on the text console as root, and run the database migration script:
+. When rebooted the {productname} spacewalk services are not running until you have migrated the PostgreSQL database to version{nbsp}13.
+. Log in on the text console as root.
+If you are upgrading from 4.1 to 4.2, run the database migration script:
++
+----
+/usr/lib/susemanager/bin/pg-migrate-12-to-13.sh
+----
+. If you are upgrading from 4.0 to 4.2, run the database migration scripts:
 +
 ----
 /usr/lib/susemanager/bin/pg-migrate-10-to-12.sh
+/usr/lib/susemanager/bin/pg-migrate-12-to-13.sh
```

Comment:
Oh, oh. I think this is not possible. Maybe we miss a migration script for this case.
To migrate a DB you need to have both installed. In this case it would require to have 10, 12 and 13 installed.
I fear 12 may not be available.

This should be checked by a developer. @jordimassaguerpla ?